features/product/managing_product_variants/configure_channel_and_currency_calculator_for_product_variant.feature

```diff
@@ -0,0 +1,23 @@
+@managing_product_variants
+Feature: Configure channel and currency calculator
```

Comment:
I guess that we have convention for naming feature files with `-ing`, so I guess it should be `Configuring...`